packages/sanity/src/core/studio/StudioLayout.tsx

```diff
@@ -173,6 +179,7 @@ export function StudioLayoutComponent() {
       {/* By using the tool name as the key on the error boundary, we force it to re-render
           when switching tools, which ensures we don't show the wrong tool having crashed */}
       <StudioErrorBoundary key={activeTool?.name} heading={`The ${activeTool?.name} tool crashed`}>
+        {process.env.NODE_ENV === 'development' && <DetectDevServerStopped />}
```

Comment:
If we only detect presence of the Vite dev server, should we add an additional check here to avoid loading and rendering `DetectDevServerStopped` if some other dev server is in use? (e.g. Studio embedded in a Next.js app).

`DetectDevServerStopped` seems like it will simply do nothing in this scenario, but omitting it altogether might avoid potential issues for non-Vite users.